src/components/List.js

```diff
@@ -13,9 +21,30 @@ const List = ({ token, results }) => {
           </NavLink>
         </>
       ) : null}
-      <ul style={{ color: 'black' }}>
+      <ul>
         {results.map(result => {
-          return <li key={result.id}>{result.name}</li>;
+          const time = Math.max(...result.purchaseDates); //pulls most recent purchase date
```

Comment:
I always forget about Math.max, such an awesome easy way to get the latest time in the `purchaseDates` array!

Reply:
I am so happy about this too!